main.py

```diff
@@ -199,29 +202,30 @@ async def register_user(user_request: PostUserRequest, db: Session = Depends(get
         location=user_request.location,
         longitude=lng_lat['lng'] if lng_lat else None,
         latitude=lng_lat['lat'] if lng_lat else None
-
     )
 
     try:
-        add_and_flush(db, user)
+        db.add(user)
```

Comment:
DB logic should live in the model, and we should ask the model to do this for us.